Submissions/infern018/csv2json.py

```diff
@@ -0,0 +1,17 @@
+import csv,json
```

Comment:
Put this script in its Task folder as stated in the README file.
Also do not create new PRs for the changes requested.
I am closing your old PR.